.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ jobs:
           type: shell
           name: Download CeKit Image Validator
           command: |
-            wget https://github.com/kiegroup/kie-cloud-tools/releases/download/1.0-SNAPSHOT/cekit-image-validator-runner.tgz
```

Comment:
I think this update was done as part of the other commit, please remove it from here and make sure this PR is rebased.

Reply:
ok